oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/SegmentAnalysisListener.java

```diff
@@ -152,6 +152,12 @@ private void appendSearchableTags(SpanObject span) {
         span.getTagsList().forEach(tag -> {
             if (searchableTagKeys.contains(tag.getKey())) {
                 final Tag spanTag = new Tag(tag.getKey(), tag.getValue());
+                if (tag.getValue().length()  > Tag.TAG_LENGTH || spanTag.toString().length() > Tag.TAG_LENGTH) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("Segment tag : {} length > : {}, dropped", spanTag, Tag.TAG_LENGTH);
```

Comment:
I think we should `warn` this, if users can't find search results for those tags, they should be notified that those are dropped instead of thinking there is no such tag at all, otherwise it's hard to diagnose why there is no result in the UI.

Reply:
Warning logs seem a little tricky. Because this kind of log would not disappear unless the user changes business codes or plugin codes. And the warning would not stop and then they would crash the log collecting system. 
I think that is not what we expected.